packages/aquarius/src/bot/plugins/starboard.js

```diff
@@ -113,10 +113,9 @@ export default async ({ aquarius, settings, analytics }) => {
         .setColor('GOLD')
         .setURL(getLink(message))
         .setDescription(message.content)
-        .addField(
-          'Source',
-          `[#${message.channel.name}](${getLink(message)})`,
-          true
+        .addFields(
+          { name: 'Source', value: `[#${message.channel.name}](${getLink(message)})`, inline: true },
+          { name: 'Stars', value: `${messageReaction.count}`, inline: true },
```

Comment:
Hm, we'd have to update this somehow right? Otherwise it'll post as 5 and just stay that way?

Reply:
Right, forgot about that.

Am I seeing it correctly that you are determining a message has already been sent to the target channel by checking the target channel and looking for an embed with the same source message link?
Looks like it when reading through lines 87 to 101.
If so, we could also check the stars amount in the embed and compare the current amount of star reactions on the message to the stars amount in the embed field.

This way we could make it remove messages which now have less than the threshold of stars, thus a way to easily remove a message from starboard again.

I hope I can read code well enough to not be talking absolute nonsense right now lol